src/ProgressBar/ProgressBar.svelte

```diff
@@ -32,6 +38,16 @@
   /** Set an id for the progress bar element */
   export let id = "ccs-" + Math.random().toString(36);
 
+  import CheckmarkFilled from "../icons/CheckmarkFilled.svelte";
+  import ErrorFilled from "../icons/ErrorFilled.svelte";
+
+  let isActiveAndDeterminate = false;
+
+  const statusIcons = {
+    error: ErrorFilled,
+    finished: CheckmarkFilled,
+  };
+
```

Comment:
Below here, please set `capped` to `0` when there is an error. Since it already uses 2 ternaries, maybe use a switch or reactive block to improve readability:

```js
let capped
$: {
  if (status === 'error' || value < 0) {
    capped = 0
  } else if (value > max) {
    capped = max
  } else {
    capped = value
  }
}
```